### PR TITLE
[ALLUXIO-2222] Change underFsType to be a String

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-2222

### CODE REVIEW

File: underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import alluxio.Constants;
 import alluxio.PropertyKey;
 import alluxio.underfs.UnderFileSystem;
+import alluxio.underfs.UnderFileSystem.SpaceType;
```
Comment (calvinjia) [inline underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java]: Is this import necessary?

File: core/common/src/main/java/alluxio/underfs/UnderFileSystem.java [hunk 2]
```diff
@@ -199,35 +199,9 @@ public static UnderFileSystem get(String path, Object ufsConf) {
...
-
-    @Override
-    public String toString() {
-      return mType;
-    }
-  }
-
-  /**
-   * @return type of concrete under filesystem implementation
+   * @return name of the under filesystem implementation
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/underfs/UnderFileSystem.java]: Maybe specify the format? (ie. all lower case, no spaces, etc.)

Comment (aaudiber) [inline core/common/src/main/java/alluxio/underfs/UnderFileSystem.java]: good idea, done

File: underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java (hunk 1 above)
Comment (aaudiber) [inline underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java]: no, removed, thanks

**Comment by aaudiber** [comment]
@calvinjia Thanks for the comments, I've updated the PR

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**